units/provisioning_create_host_test.go

```diff
@@ -33,61 +31,18 @@ func TestProvisioningCreateHostJob(t *testing.T) {
 			assert.Equal(t, h.Id, hostCreateJob.HostID)
 			assert.Equal(t, *h, *hostCreateJob.host)
 		},
-		"SucceedsForAMICreateHost": func(ctx context.Context, t *testing.T, env *mock.Environment, mgr *jmock.Manager, h *host.Host) {
+		"SucceedsForAMICreateHost": func(ctx context.Context, t *testing.T, env *mock.Environment, h *host.Host) {
```

Comment:
nit: Test name should probably just be something like `SucceedsForHostCreate` since it's not specifically testing `host.create` with an AMI anymore.